portal-impl/src/com/liferay/portlet/documentlibrary/util/DLImpl.java

```diff
@@ -864,22 +864,23 @@ public String getTitleWithExtension(String title, String extension) {
 	public String getUniqueFileName(
```

Comment:
Should we do the same in  [DefaultUniqueFileNameProvider](https://github.com/liferay-lima/liferay-portal/blob/091bf6a11e607f516e7852e792d94d30387d5c55/modules/apps/upload/upload-web/src/main/java/com/liferay/upload/web/internal/DefaultUniqueFileNameProvider.java)?

Reply:
Yes, I have a task on my notebook to review Google Drive and OneDrive behaviour and it is when this must be changed, on my opinion

Reply:
Well, better create a story for that, Thanks for pointing at it!